components/menu/index.css

```diff
@@ -11,849 +11,945 @@ governing permissions and limitations under the License.
+	--spectrum-menu-drillin-icon-width: var(--spectrum-chevron-icon-size-75);
+
+	--spectrum-menu-linkout-icon-height: var(--spectrum-link-out-icon-size-75);
+	--spectrum-menu-linkout-icon-width: var(--spectrum-link-out-icon-size-75);
+
+	--spectrum-menu-back-icon-margin: var(--spectrum-navigational-indicator-top-to-back-icon-small);
 }
 
 .spectrum-Menu--sizeL {
-  --spectrum-menu-item-min-height: var(--spectrum-component-height-200);
-  --spectrum-menu-item-icon-height: var(--spectrum-workflow-icon-size-200);
-  --spectrum-menu-item-icon-width: var(--spectrum-workflow-icon-size-200);
-  --spectrum-menu-item-label-font-size: var(--spectrum-font-size-200);
-  --spectrum-menu-item-label-text-to-visual: var(--spectrum-text-to-visual-200);
+	--spectrum-menu-item-min-height: var(--spectrum-component-height-200);
+	--spectrum-menu-item-corner-rounding: var(--spectrum-corner-radius-600);
+	--spectrum-menu-item-icon-height: var(--spectrum-workflow-icon-size-200);
+	--spectrum-menu-item-icon-width: var(--spectrum-workflow-icon-size-200);
+	--spectrum-menu-item-thumbnail-height: var(--spectrum-thumbnail-size-800);
+	--spectrum-menu-item-thumbnail-width: var(--spectrum-thumbnail-size-800);
+	--spectrum-menu-item-label-font-size: var(--spectrum-font-size-200);
+	--spectrum-menu-item-label-text-to-visual: var(--spectrum-text-to-visual-200);
+
+	--spectrum-menu-item-label-inline-edge-to-content: var(--spectrum-component-edge-to-text-200);
+	--spectrum-menu-item-top-edge-to-text: var(--spectrum-component-top-to-text-200);
+	--spectrum-menu-item-bottom-edge-to-text: var(--spectrum-component-bottom-to-text-200);
+	--spectrum-menu-item-top-to-icon: var(--spectrum-component-top-to-workflow-icon-200);
+	--spectrum-menu-item-top-to-drillin: var(--spectrum-menu-item-top-to-disclosure-icon-large);
+	--spectrum-menu-item-top-to-thumbnail: var(--spectrum-menu-item-top-to-thumbnail-large);
+	--spectrum-menu-item-thumbnail-to-label: var(--spectrum-text-to-visual-300);
+
+	--spectrum-menu-item-text-to-control: var(--spectrum-text-to-control-200);
 
-  --spectrum-menu-item-label-inline-edge-to-content: var(--spectrum-component-edge-to-text-200);
-  --spectrum-menu-item-top-edge-to-text: var(--spectrum-component-top-to-text-200);
-  --spectrum-menu-item-bottom-edge-to-text: var(--spectrum-component-bottom-to-text-200);
+	--spectrum-menu-item-description-font-size: var(--spectrum-font-size-100);
 
-  --spectrum-menu-item-text-to-control: var(--spectrum-text-to-control-200);
+	--spectrum-menu-section-header-font-size: var(--spectrum-font-size-200);
+	--spectrum-menu-section-header-min-width: var(--spectrum-component-height-200);
+	--spectrum-menu-section-description-font-size: var(--spectrum-font-size-100);
+	--spectrum-menu-section-header-to-description: var(--spectrum-menu-section-header-to-description-large);
+	--spectrum-menu-item-label-to-description-spacing: var(--spectrum-menu-item-label-to-description-large);
```

Comment:
What should we do with the word "spacing" in these few properties? Should `--spectrum-menu-section-header-to-description` be `--spectrum-menu-section-header-to-description-spacing` (adding `-spacing`)
OR
Should --`spectrum-menu-item-label-to-description-spacing` be `--spectrum-menu-item-label-to-description` (without `-spacing`)? 

I like the idea of having spacing in the property name, but none of the other properties have that so I'm cool with removing it from `label-to-description`. What do you think?

Reply:
Good point, it'd be good if they matched. Since we already had `--spectrum-menu-item-label-to-description-spacing` and I'm hesitant to change it, I'll update `--spectrum-menu-section-header-to-description` to be `--spectrum-menu-section-header-to-description-spacing` instead.